llvm/test/CodeGen/NVPTX/shift-opt.ll

```diff
@@ -0,0 +1,40 @@
+; RUN: llc < %s -mtriple=nvptx64 | FileCheck %s
+
+define i64 @test1(i64 %x, i32 %y) {
```

Comment:
Descriptive function name?

Reply:
I reworded test names. Not sure they've become much more descriptive. But significantly better than numbered version.